src/phylum/ci/ci_base.py

```diff
@@ -449,10 +449,15 @@ def _get_project_threshold_info(self, project_thresholds: dict, threshold_type:
         if threshold is None:
             risk_domain: RiskDomain = PROJECT_THRESHOLD_OPTIONS.get(threshold_type, RiskDomain("", "", ""))
             threshold = project_thresholds.get(risk_domain.project_name)
-            req_src = "project setting"
+            req_src = "project per-axis threshold"
             if threshold is None:
                 threshold = 1.0
                 req_src = "N/A (fail safe)"
+
+            total_threshold = project_thresholds.get("total", 0.0)
+            if threshold < total_threshold:
```

Comment:
```suggestion
            if threshold == 0:
```
An example of where the original line would produce unexpected results:

* The project per-axis threshold setting for `license` is enabled and set to 40
* The total project threshold setting is at 60
* The `--lic-threshold` option was NOT provided to `phylum-ci`

The result in this scenario setup is that the license threshold used to report on individual packages in the PR would be 60 instead of the expected/set 40.

Reply:
I understand the reasoning better now...thanks for the clarification.